server/notification-providers/teams.js

```diff
@@ -57,41 +60,114 @@ class Teams extends NotificationProvider {
+                                                "verticalContentAlignment": "Center",
+                                                "items": [
+                                                    {
+                                                        "type": "Image",
+                                                        "width": "32px",
+                                                        "style": "Person",
+                                                        "url": "https://raw.githubusercontent.com/louislam/uptime-kuma/master/public/icon.png",
+                                                        "altText": "Uptime Kuma Logo"
+                                                    }
+                                                ]
+                                            },
+                                            {
+                                                "type": "Column",
+                                                "width": "stretch",
+                                                "items": [
+                                                    {
+                                                        "type": "TextBlock",
+                                                        "size": "Medium",
+                                                        "weight": "Bolder",
+                                                        "text": headerMessage,
+                                                    },
+                                                    {
+                                                        "type": "TextBlock",
+                                                        "size": "Small",
+                                                        "weight": "Default",
+                                                        "text": "Uptime Kuma Alert",
+                                                        "isSubtle": true,
+                                                        "spacing": "None"
+                                                    }
+                                                ]
+                                            }
+                                        ]
+                                    }
+                                ]
+                            },
+                            {
+                                "type": "FactSet",
+                                "separator": true,
+                                "facts": facts
+                            }
+                        ],
+                        "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+                        "version": "1.5"
+                    }
+                }
+            ]
         };
+
+        if (dashboardUrl) {
+            payload.attachments.forEach(element => {
+                element.content.push({
+                    "type": "ActionSet",
+                    "actions": [
+                        {
```

Comment:
In terms of actions:
Do you think the URL should be clickable text or an action? (open to both, would just know other peoples feelings on this)

Reply:
I'm not completely sure, what you mean. Another Action-Button for the Monitor-URL?

Reply:
Yes, the alternative would be another action button. Example:
![image](https://github.com/louislam/uptime-kuma/assets/26258709/71cb7897-c3c8-421f-a96a-852012d73a3b)

Reply:
Sure, that's possible. Time is now added, too.
<img width="466" alt="image" src="https://github.com/louislam/uptime-kuma/assets/29057942/59973bcf-fc44-44a0-9ef7-fad90c4fd786">